Classes/common/CDTDatastore.h

```diff
@@ -70,7 +71,7 @@ extern NSString *const CDTDatastoreChangeNotification;
 
 + (NSString *)versionString;
 
-- (id)initWithDatabase:(TD_Database *)database;
+- (id)initWithManager:(CDTDatastoreManager *) manager Database:(TD_Database *)database;
```

Comment:
I think the first `D` in `Database:` should be lowercase. Rename `initWithManager:Database:` to `initWithManager:database:`